web-components/src/components/editable-textfield/EditableTextfield.ts

```diff
@@ -230,7 +230,6 @@ export class EditableTextfield extends LitElement {
           this.handleKeydown(e);
         }}
         aria-invalid=${this.alert ? "true" : "false"}
-        aria-errormessage=${ifDefined(this.message?.message)}
```

Comment:
@futurethang please review. what does removal of this aria-label achieves ?

Reply:
@yananym Unclear as to why this is removed, though perhaps the `aria-errormessage` is better to affix to the `<md-help-text>` that `messagesTemplate()` returns. That is not currently there, but could be added, otherwise we should restore this deleted line. Thoughts?

Reply:
Added to <md-help-text>, because if add this attribute to textfield we get critical accessibility error:
`Critical
aria-errormessage values `That, is, not, a, valid, input.` must use a technique to announce the message (e.g., aria-live, aria-describedby, role=alert, etc.)`

But if we add this attribute to the <md-help-text> block, we can avoid this error.